src/Bicep.Core/TypeSystem/Az/AzResourceTypeLoaderFactory.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Bicep.Types.Az;
+using Bicep.Core.Syntax;
+using System.IO;
+using Bicep.Core.Registry.Oci;
+using Newtonsoft.Json;
+using Bicep.Core.Features;
+
+namespace Bicep.Core.TypeSystem.Az
+{
+    public class AzResourceTypeLoaderFactory : IAzResourceTypeLoaderFactory
+    {
+        private readonly IFeatureProviderFactory featureProviderFactory;
+
+        private Dictionary<string, IAzResourceTypeLoader> resourceTypeLoaders = new(){
```

Comment:
nit: Formatting.

Reply:
@shenglol, what is the desired formatting ?

Reply:
Try Format Document context menu.

Reply:
I did and it does nothing for me. Maybe its my formatter settings?

Reply:
added a newline on the curly bracket based on @majastrz input :). Thank you @majastrz :)